server/setup.py

```diff
@@ -51,12 +51,17 @@
     ],
     extras_require={
         'onnx': [
-            'onnxruntime',
+            'onnxruntime==1.13.1',
```

Comment:
I would prefer to use `onnxruntime<=1.13.1`

Reply:
let me do some permutations to pass the ci first 😅